src/main/java/Duke.java

```diff
@@ -1,10 +1,89 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.util.Scanner;
+
+public class Duke {
+
+    private static void printAddTaskConfirmation(Task task, int numTasks) {
+        String line = "____________________________________________________________";
+        System.out.println(line);
+        System.out.println("Got it. I've added this task:\n" + task + "\nNow you have " + numTasks + " tasks in the list.");
+        System.out.println(line);
+    }
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        //System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        
+        String line = "____________________________________________________________";
+
+        System.out.println(line);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        System.out.println(line);
+        
+        Task[] tasks = new Task[100];
+        int numTasks = 0;
+
+        while (true) {
+            System.out.println();
+            String cmd = in.nextLine();
+            String[] splitIntoArgs = cmd.split(" ", 2);
+
+            if (cmd.equals("bye")) {
+                System.out.println(line);
+                System.out.println("Bye. Hope to see you again soon!");
+                System.out.println(line);
+                break;
+            } else if (cmd.equals("list")) {
+                System.out.println(line);
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < numTasks; i++) {
+                    System.out.println(Integer.toString(i+1) + "." + tasks[i]);
+                }
+                System.out.println(line);
+            } else if (splitIntoArgs[0].equals("mark")) {
+                int toMark = Integer.parseInt(splitIntoArgs[1]) - 1;

```

Comment:
It may be hard for someone to understand what is being done in this step, so you could add a comment explaining or extract it as a method (eg. getIndex()) for readability